content/blog/2020-call-for-proposals-announcement.md

```diff
@@ -0,0 +1,74 @@
+Title: Calling all Pythonistas in the world, Call for Proposals for PyCon India 2020 is now open!
```

Comment:
Can we add Online in the title to mention PyCon India 2020 is going online ? THis is also an announcement post.

Reply:
I was actually thinking, why not do an announcement post as well? 
The conf going online is a significant development. Can't hurt right.

Reply:
Also, didn't someone mention on one call regarding about not adding online to the branding much. Adding it the title just makes it a little odd. 

IMHO we can have only one in the title, either we can have `online` or `2020`?

Reply:
I am ok with a separate announcement blog. But does it matter now since we have already made a Twitter announcement ?